<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer</class>
 <widget class="QDialog" name="customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA CUSTOMER</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>160</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="namaCustLabel">
      <property name="text">
       <string>Nama Cust</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="namaCustLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="siupCustLabel">
      <property name="text">
       <string>Siup Cust</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="siupCustLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="alamatCustLabel">
      <property name="text">
       <string>Alamat Cust</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="alamatCustLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="email1Label">
      <property name="text">
       <string>Email 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="email1LineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="telp1Label">
      <property name="text">
       <string>Telp 1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="telp1LineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>160</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="fax1Label">
      <property name="text">
       <string>Fax 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="fax1LineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="cPCustLabel">
      <property name="text">
       <string>CP Cust</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="cPCustLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="email2Label">
      <property name="text">
       <string>Email 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="email2LineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telp2Label">
      <property name="text">
       <string>Telp 2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="telp2LineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="iDUserLabel">
      <property name="text">
       <string>ID User</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="iDUserLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
